src/Http/Http.Abstractions/src/Metadata/IStatusCodePagesMetadata.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Http.Metadata;
+
+/// <summary>
+/// Defines a contract used to specify metadata for skipping the StatusCodePage
+/// middleware in <see cref="Endpoint.Metadata"/>.
```

Comment:
Should we drop the `in Endpoint.Metadata`? I don't know if it helps clarify things?

Reply:
I like the `in Endpoint.Metadata`. What's wrong with it? It could point the curious to API docs for endpoint routing.